lib/modules/manager/asdf/upgradeable-tooling.ts

```diff
@@ -261,6 +261,26 @@ export const upgradeableTooling: Record<string, ToolingDefinition> = {
           currentValue: adoptOpenJreMatches.version,
         };
       }
+      const semeruJdkMatches = version.match(
+        /^semeru-openj9-(?<version>\d\S+)_openj9-(?<openj9>\d\S+)/
```

Comment:
```suggestion
        /^semeru-openj9-(?<version>\d\S+)_openj9-(?<openj9>\d\S+)/
```
this will create invalid PR's when openj9 version changes.

Reply:
Ouch, yes that's unfortunate.

Do you have any ideas how we might solve this?

Reply:
Here's what Scoop does, could we do something similar?

https://github.com/ScoopInstaller/Java/blob/master/bucket/semeru-lts-jre.json#L18-L19

I.e. make use of:
- https://github.com/ibmruntimes/semeru17-binaries/releases
- https://github.com/ibmruntimes/semeru11-binaries/releases
- https://github.com/ibmruntimes/semeru8-binaries/releases

Reply:
this isn't easily solvable, we should revert this PR until we've a proper working datasource / versioning for this.